openbb_platform/providers/oecd/openbb_oecd/models/composite_leading_indicator.py

```diff
@@ -84,13 +84,22 @@ def extract_data(
         **kwargs: Any,
     ) -> Dict:
```

Comment:
Here the output is a `List[Dict]`, the mismatch will trigger a test failure.  The same applies to the Input of the `transform_data` method.

Reply:
should be all set now